lymproblems/assignsudo.py

```diff
@@ -0,0 +1,19 @@
+# Assign sudo-user/superuser privileges to a current user "lym".
+
+import os
+import sys
+from .utils import system, success, fail, find_path_data
+
+
+def setup():
+    "Setup problemname"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify problemname"
+    user = "lym"
+
+    if user not in os.popen("getent group sudo").read():
```

Comment:
Please use the `system` function than using os.popen directly.

Also, the solution is wrong. We don't add people to `sudo` to get sudo access.

Reply:
I will re-work on this. Thank you for the reviews/suggestion.

Reply:
@kushaldas, please review.